src/Methods/DebitPaymentMethod.php

```diff
@@ -257,4 +321,63 @@ public function canHandleSubscriptions():bool
     {
         return true;
     }
+
+    /**
+     * @param int $customerId
+     * @return bool
+     */
+    private function isGuest($customerId)
+    {
+        return !$this->accountService->getIsAccountLoggedIn() && (int)$customerId === 0;
+    }
+
+    /**
+     * @param Contact $contact
+     * @return bool
+     */
+    private function isExplicitlyAllowedForThisCustomer(Contact $contact)
+    {
+        if(!$this->isGuest($contact->id)) {
+
+            if (!is_null($contact) && $contact instanceof Contact) {
```

Comment:
In PHP 7.3 this can never happen! 
If the method is called with `$contact === null`, then a Fatal Error occures. I
```
Fatal error: Uncaught TypeError: Argument 1 passed to isExplicitlyAllowedForThisCustomer() must be an instance of Contact, null given
```

If `null` should be a valid value for the method, you have to declare `null` as default value:
```php
private function isExplicitlyAllowedForThisCustomer(Contact $contact = null)
```